File: app/src/main/java/sakaitakao/android/permissionsearch/activity/AboutActivity.java
package sakaitakao.android.permissionsearch.activity;

import sakaitakao.android.permissionsearch.R;
import android.app.Activity;
import android.os.Bundle;

/**
 * バージョン情報のアクティビティ
 * 
 * @author takao
 * 
 */
public class AboutActivity extends Activity {
	/** Called when the activity is first created. */
	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.about);
	}
}

File: app/src/main/kotlin/sakaitakao/android/permissionsearch/activity/AboutActivity.kt
package sakaitakao.android.permissionsearch.activity

import android.app.Activity
import android.os.Bundle
import sakaitakao.android.permissionsearch.R

/**
 * バージョン情報のアクティビティ

 * @author takao
 */
class AboutActivity : Activity() {
    /** Called when the activity is first created.  */
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.about)
    }
}